build.sbt

```diff
@@ -152,7 +176,13 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform, NativePlatform)
   .jvmSettings(
     mainClass in GraalVMNativeImage := Some("metaconfig.tests.ExampleMain"),
     sources.in(Compile, doc) := Seq.empty,
-    libraryDependencies += "com.github.alexarchambault" %%% "scalacheck-shapeless_1.15" % "1.3.0",
+    libraryDependencies ++= {
+      if (scalaVersion.value.startsWith("2."))
+        Seq(
+          "com.github.alexarchambault" %%% "scalacheck-shapeless_1.15" % "1.3.0"
```

Comment:
It's fine to move those test suites to Scala 2-only directories

Reply:
I've moved the codec roundtrip test there, this is just to prevent `+test` trying to resolve non-existent dependency